src/core/typing.ts

```diff
@@ -138,22 +138,10 @@ export class DefaultTyping
    * @param logger An instance of the Logger.
    * @param initAfter A promise that is awaited before creating any channels.
```

Comment:
initAfter can be removed here and line 152

Reply:
Done